src/test/java/OsduSimpleClientTest.java

```diff
@@ -0,0 +1,59 @@
+import java.io.IOException;
+
+import org.json.JSONException;
+import org.testng.annotations.*;
+import pariveda.osdu.OsduSimpleClient;
+
+public class OsduSimpleClientTest {
+
+    private OsduSimpleClient simpleClient;
+    private String dataPartition;
+    private String token;
+    private String successUrl;
+    private String successContentType;
+ 
+    @BeforeClass
+    public void setUp() {
+      dataPartition = "opendes";
+      token = "abcde";
+      successUrl = "https://server.test.com";
+      successContentType = "application/json";
+      simpleClient = new OsduSimpleClient(successUrl, dataPartition, successContentType, token);
+    }
+    
+    @Test(groups = { "integration" }, enabled=false ) // Disabled for now; need to refactor to leverage env variables for valid tokens
+    public void expectedSearchSuccess() {
+      String query = "{ kind: \"*:*:*:*\", limit: 1 }";
+      try {
+        simpleClient.Search(query);
+        System.out.println("expectedSearchSuccess: Search query is valid.");
+      } catch (Exception e) {
+          org.testng.Assert.fail("unexpected exception", e);
+      }
+    }
+    
+    @Test(groups = { "validation" }, 
+    expectedExceptions = JSONException.class) 
+    public void expectedSearchJSONError() {
+        String query = "test123";
+        try {
+            simpleClient.Search(query);
+        }
+        catch (IOException e) {
+            System.out.println("expectedSearchError: caught unexpected IOException:\n" + e.getMessage());
```

Comment:
Messaging is a bit odd for an unexpected exception when we do expect it in this case

Reply:
it's a bit misleading, because we're actually expecting a JSONException in this test; the IOException is needed because of the way Search is built to directly call OSDU APIs, so we need to catch the networking IOException in order for the test to be valid Java. I'd like to take on a larger refactor to decouple our client layer into client & service layers like you have in the nodejs version, which should enable us to remove this